src/tokenizer/index.ts

```diff
@@ -44,29 +44,28 @@ function normalizeToken(token: string, language: Language, tokenizerConfig: Toke
 
   if (normalizationCache.has(key)) {
     return normalizationCache.get(key)!;
-  } else {
-    // Check if stop-words removal is enabled
-    if (tokenizerConfig?.enableStopWords) {
-      // Remove stop-words
-      if ((tokenizerConfig?.customStopWords as string[]).includes(token)) {
-        const token = "";
-        normalizationCache.set(key, token);
-        return token;
-      }
+  }
+  // Check if stop-words removal is enabled
+  if (tokenizerConfig?.enableStopWords) {
+    // Remove stop-words
+    if ((tokenizerConfig?.customStopWords as string[]).includes(token)) {
```

Comment:
By the way, `Array.prototype.includes` is slow too. But it's out of the topic of this PR.

Reply:
@RafaelGSS true, but the the`customStopWords ` array is typically pretty small, so it wouldn't affect the performance that much